### PR TITLE
Remove query_auth from the specification per MSC2451.

### PR DESCRIPTION
**Pull request by clokep**
This is per MSC #2451 ([rendered version](https://github.com/matrix-org/matrix-doc/blob/master/proposals/2451-remove-query_auth-federation-endpoint.md)).

Synapse v1.12.0 removed this endpoint in matrix-org/synapse#7026.

### CODE REVIEW

**Comment by clokep** [comment]
Oops! Added!

File: changelogs/server_server/newsfragments/2470.removal [hunk 1]
```diff
@@ -0,0 +1 @@
+Remove the unused ``query_auth`` API per [MSC2451](https://github.com/matrix-org/matrix-doc/pull/2451).
```
Comment (turt2live) [inline changelogs/server_server/newsfragments/2470.removal]: We use RST:
```suggestion
Remove the unused ``query_auth`` API per `MSC2451 <https://github.com/matrix-org/matrix-doc/pull/2451>`_.
```

**Comment by clokep** [comment]
> so we don't have removal as a changelog type, but this sort of change applies to no other changelog type. I'll merge this and get a removal type added.

Thanks, was just going off of what Synapse has.

File: changelogs/server_server/newsfragments/2470.removal (hunk 1 above)
Comment (clokep) [reply changelogs/server_server/newsfragments/2470.removal]: Ah, OK. Thanks!

**Pull request merged**